### PR TITLE
🎨 AccountV1 review fixes

### PR DESCRIPTION
**Pull request by j-vp**

### CODE REVIEW

File: test/fuzz/accounts/AccountV1/TransferOwnership.fuzz.t.sol [hunk 1]
```diff
@@ -27,24 +27,13 @@ contract TransferOwnership_AccountV1_Fuzz_Test is AccountV1_Fuzz_Test {
...
         vm.startPrank(sender);
-        vm.expectRevert(AccountErrors.Only_Factory.selector);
+        vm.expectRevert(AccountErrors.OnlyFactory.selector);
         accountExtension.transferOwnership(to);
         vm.stopPrank();
 
         assertEq(users.accountOwner, accountExtension.owner());
     }
 
-    function testFuzz_Revert_transferOwnership_InvalidRecipient() public {
```
Comment (Thomas-Smets) [inline test/fuzz/accounts/AccountV1/TransferOwnership.fuzz.t.sol:37]: Is it tested on the Factory?

File: src/AccountV1.sol [hunk 2]
```diff
@@ -1085,7 +1063,7 @@ contract AccountV1 is AccountStorageV1, IAccount {
         for (; i < erc20StoredLength;) {
             cacheAddr = erc20Stored[i];
             assetAddresses[i] = cacheAddr;
-            //assetIds[i] = 0; //gas: no need to store 0, index will continue anyway.
+            // assetIds[i] = 0; // gas: no need to store 0, index will continue anyway.
```
Comment (Thomas-Smets) [inline src/AccountV1.sol:1066]:
```suggestion
            // assetIds[i] = 0; // Gas: no need to store 0, index will continue anyway.
```

File: src/AccountV1.sol [hunk 3]
```diff
@@ -634,88 +600,87 @@ contract AccountV1 is AccountStorageV1, IAccount {
      * @param assetAddresses Array of the contract addresses of the assets.
      * @param assetIds Array of the IDs of the assets.
      * @param assetAmounts Array with the amounts of the assets.
-     * @param from The address to withdraw the assets from.
+     * @param from The address to deposit the assets from.
```
Comment (Thomas-Smets) [inline src/AccountV1.sol:603]: Not entirely correct, on a deposit you indeed withdraw assets from the owner, better rephrase diferently.

File: src/AccountV1.sol [hunk 4]
```diff
@@ -11,22 +11,21 @@ import { ICreditor } from "./interfaces/ICreditor.sol";
...
 import { AccountStorageV1 } from "./AccountStorageV1.sol";
 import { RiskModule } from "./RiskModule.sol";
 import { AccountErrors } from "./libraries/Errors.sol";
 
 /**
- * @title Acadia Accounts.
+ * @title Arcadia Accounts
  * @author Pragma Labs
- * @notice Arcadia Accounts are smart contracts that act as onchain, decentralized and composable margin accounts.
+ * @notice Arcadia Accounts are smart contracts that act as on-chain, decentralized and composable margin accounts.
```
Comment (zekiblue) [inline src/AccountV1.sol:22]: i thought convention is onchain, my fixes was towards onchain

Comment (j-vp) [reply src/AccountV1.sol:22]: you're right, edited

File: src/AccountV1.sol [hunk 5]
```diff
@@ -11,22 +11,21 @@ import { ICreditor } from "./interfaces/ICreditor.sol";
...
  * They provide individuals, DAOs, and other protocols with a simple and flexible way to deposit and manage multiple assets as collateral.
  * The total combination of assets can be used as margin to back liabilities issued by any financial protocol (lending, leverage, futures...).
- * @dev Users can use this Account to deposit assets (ERC20, ERC721, ERC1155, ...).
- * The Account will denominate all the pooled assets into one baseCurrency (one unit of account, like usd or eth).
- * An increase of value of one asset will offset a decrease in value of another asset.
+ * @dev Users can use this Account to deposit assets (fungible, non-fungible, LP positions, yiel bearing assets...).
+ * The Account will denominate all the deposited assets into one baseCurrency (one unit of account, like usd or eth).
```
Comment (zekiblue) [inline src/AccountV1.sol:26]:
```suggestion
 * The Account will denominate all the deposited assets into one baseCurrency (one unit of account, like USD or ETH).
```

File: src/AccountV1.sol (hunk 3 above)
Comment (zekiblue) [reply src/AccountV1.sol:603]: "The address where the assets are originated from" ?

File: src/AccountV1.sol [hunk 6]
```diff
@@ -70,47 +69,44 @@ contract AccountV1 is AccountStorageV1, IAccount {
...
-        if (msg.sender != IRegistry(registry).FACTORY()) revert AccountErrors.Only_Factory();
+        if (msg.sender != IRegistry(registry).FACTORY()) revert AccountErrors.OnlyFactory();
         _;
     }
 
     /**
-     * @dev Throws if called by any account other than the owner.
+     * @dev Throws if called by any address other than the owner.
      */
     modifier onlyOwner() {
```
Comment (acollette) [inline src/AccountV1.sol:89]: As this one is used 6 times, there's an optimization to use an internal function for the require statement.
Would save up to 30_000 gas on each deployment.

Comment (j-vp) [reply src/AccountV1.sol:89]: on each implementation deployment right?

Comment (acollette) [reply src/AccountV1.sol:89]: yes

Comment (j-vp) [reply src/AccountV1.sol:89]: will keep it as is for now, can be other pr but low prio

**Review by Thomas-Smets — APPROVED** [review]
gtg

**Pull request merged**